README.md

```diff
@@ -25,117 +27,108 @@ We use the Docker's building system and convert it at the end to a disk image th
 ##### 1. Making the environment
 
 ```console
-root@yourmachine> scw run --name="buildcowsay" builder
-Welcome to the image to build other images on Scaleway' C1.
-
- * Kernel:           GNU/Linux 4.0.5-235 armv7l
- * Distribution:     An image to build other images (2015-08-06) on Ubuntu 15.04
- * Internal ip:      X.X.X.X
- * External ip:      X.X.X.X
- * Disk /dev/nbd0:   buildcowsay-scw-image-builder-1.0-2015-08-06_15:20 (l_ssd 50G)
- * Uptime:           19:44:48 up 1 min,  0 users,  load average: 0.44, 0.13, 0.05
-
-Links
- * Documentation:    https://scaleway.com/docs
- * Community:        https://community.scaleway.com
- * Image source:     https://github.com/scaleway/image-tools/tree/master/image-builder
-
-Docker 1.7.1 is running using the 'aufs' storage driver.
-Installed tools: docker-compose, nsenter, gosu and pipework are installed.
-Getting started with Docker on C1: https://community.cloud.online.net/t/383?u=manfred.
+root@yourmachine> scw run --name="buildcowsay" image-builder
+               _
+ ___  ___ __ _| | _____      ____ _ _   _
+/ __|/ __/ _` | |/ _ \ \ /\ / / _` | | | |
+\__ \ (_| (_| | |  __/\ V  V / (_| | |_| |
+|___/\___\__,_|_|\___| \_/\_/ \__,_|\__, |
+                                    |___/
+...
 
 *****************************************************************************
 
    Welcome on the image-builder.
-   Here, you will be able to craft your own images.
+   Here, you'll be able to craft your own images.
 
-   To configure your environment please run:
+   To configure your environment run:
 
    $> image-builder-configure
 
 *****************************************************************************
 
-The programs included with the Ubuntu system are free software;
-the exact distribution terms for each program are described in the
-individual files in /usr/share/doc/*/copyright.
-
-Ubuntu comes with ABSOLUTELY NO WARRANTY, to the extent permitted by
-applicable law.
+...
 
-root@buildcowsay> image-builder-configure
+root@buildcowsay:~# image-builder-configure
 Login (cloud.scaleway.com):                     # yourmail
 Password:                                       # yourpassword
-root@buildcowsay> mkdir vivid-cowsay
-root@buildcowsay> cp Makefile vivid-cowsay
-root@buildcowsay> cd vivid-cowsay
-root@buildcowsay> touch Dockerfile
-root@buildcowsay> ls -l
+root@buildcowsay:~# mkdir vivid-cowsay
```

Comment:
remove the vivid- everywhere, it's more clear to talk about an application instead of an application with a specific context